hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/realtime/HoodieOrcRealtimeInputFormat.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2016 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.hoodie.hadoop.realtime;
+
+import com.uber.hoodie.common.model.HoodieRecord;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.hadoop.conf.Configurable;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.ql.io.orc.OrcInputFormat;
+import org.apache.hadoop.hive.serde2.ColumnProjectionUtils;
+
+/**
+ * ORC Input Format, that provides a real-time view of data in a Hoodie dataset
+ */
+public class HoodieOrcRealtimeInputFormat extends OrcInputFormat implements Configurable {
```

Comment:
can we create two input formats like we do for Parquet.. 

HoodieOrcInputFormat and HoodieOrcRealtimeInputFormat to match HoodieInputFormat and HoodieRealtimeInputFormat?